### PR TITLE
ci: boost demo build

### PR DESCRIPTION
**Pull request by quentinderoubaix**

### CODE REVIEW

File: demo/package.json [hunk 1]
```diff
@@ -91,8 +91,8 @@
 		"@sveltejs/vite-plugin-svelte": "^3.0.2",
 		"@types/dom-view-transitions": "^1.0.4",
 		"bootstrap-icons": "^1.11.3",
+		"compare-versions": "^6.1.0",
```
Comment (andreascorti) [inline demo/package.json:94]: Shouldn't we use it also in our scripts and remove completely the dependency to `semver` then?

**Review by andreascorti — COMMENTED** [review]
Hey @quentinderoubaix, I don't understand why we use 2 libraries for the same thing (semver / compare-versions).

File: scripts/versionDocFolder.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const {parse} = require('semver');
+const regexMajorMinor = /^(\d+)\.(\d+)/;
```
Comment (andreascorti) [inline scripts/versionDocFolder.js:1]: had a look quickly to the library, shouldn't you use `validateAndParse`? https://github.com/omichelsen/compare-versions/blob/main/src/utils.ts#L9C14-L9C30

Comment (andreascorti) [reply scripts/versionDocFolder.js:1]: my bad, they do not export utils

**Review by andreascorti — APPROVED** [review]

**Comment by quentinderoubaix** [comment]
thx for the review @andreascorti !

**Pull request merged**